package org.pessoal.desafiosiad.exceptions;

public class CPFInvalidoException extends Exception {

    public CPFInvalidoException() {
        super("CPF invalido");
    }
}
